mrmShared/linux/uio_mrf.c

```diff
@@ -123,7 +123,7 @@ int mrf_mmap_physical(struct uio_info *info, struct vm_area_struct *vma)
         return -EINVAL;
     }
 
-#if (LINUX_VERSION_CODE >= KERNEL_VERSION(6, 3, 0))
+#if (LINUX_VERSION_CODE >= KERNEL_VERSION(6, 3, 0)) || (defined(RHEL_RELEASE_CODE) && (RHEL_RELEASE_CODE >= 906))
```

Comment:
Is `906` meant to be `0x906`, or better yet `RHEL_RELEASE_VERSION(9,6)`?

The situation seems to me confusing enough to merit a comment describing what we think happened.

```c
/* vm_flags_set() function introduced in upstream Linux 6.3.
 * Backported to RHEL 9.6 ???
 */
```

Has anyone found a reference for how `RHEL_RELEASE_VERSION` progresses wrt. distribution versions?  Or a summary of how it diverges for the various RHEL derivatives?

Reply:
@mdavidsaver  I haven't been using those distros but it looks like ```Rocky Linux``` and ```AlmaLinux``` don't backport but ```CentOS Stream``` does. All those 3 distros use the same macros.
I haven't found an easy solution but [universal solution](https://github.com/jerzyjamroz/mrfioc2/commits/c9test), so I am waiting for eventual proposals.

Reply:
> Is `906` meant to be `0x906`, or better yet `RHEL_RELEASE_VERSION(9,6)`?

To be clear `906 == 0x38a == RHEL_RELEASE_VERSION(3, 138)` is not what is intended.  This is a bug.  The condition needs to be on `0x906` or `RHEL_RELEASE_VERSION(9,6)`.

Reply:
@mdavidsaver , I started a thread with the Centos Packaging Team [discussion.fedoraproject](https://discussion.fedoraproject.org/t/centos-stream-9-backports-but-almalinux-and-rocky-linux-dont/135492). They wrote some proposals.